plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/cwqTab/nodes/QServiceNode.kt

```diff
@@ -75,7 +75,7 @@ class QRootNode(private val nodeProject: Project) : AbstractTreeNode<String>(nod
         }
         return if (groupId == Q_SIGNED_IN_ACTION_GROUP) {
             val signedInNodes = childNodes + OpenCodeReferenceNode(nodeProject)
-            if (connection.connectionType == ActiveConnectionType.IAM_IDC && isCodeModernizerAvailable(project)) {
+            if (connection.connectionType == ActiveConnectionType.IAM_IDC && isCodeTransformAvailable(project)) {
```

Comment:
Question: We might have a re-usable method to use for auth here. Or need to create one. I see this is `CodeModernizerManager.kt`

```
        val connection = checkBearerConnectionValidity(project, BearerTokenFeatureSet.Q)
        if (!(connection.connectionType == ActiveConnectionType.IAM_IDC && connection is ActiveConnection.ValidBearer)) {
            return ValidationResult(
                false,
                message("codemodernizer.notification.warn.invalid_project.description.reason.not_logged_in"),
                InvalidTelemetryReason(
                    CodeTransformPreValidationError.NonSsoLogin
                )
            )
        }
        ```

Reply:
It looks like we might have `isCodeModernizerAvailable()` in the `CodeTransformUtils.kt` file

Reply:
Changed to using isCodeTransformAvailable